ssl-config/src/main/java/com/palantir/remoting/ssl/SslConfiguration.java

```diff
@@ -28,18 +30,44 @@
 @Value.Style(visibility = ImplementationVisibility.PACKAGE)
 public abstract class SslConfiguration {
 
-    @Value.Parameter
-    public abstract TrustStoreConfiguration trust();
+    public abstract URI trustStorePath();
```

Comment:
Fair point. I'm okay with `trustStoreUri` and `keyStoreUri` to keep same general format but to make the type more clear from the param name.